### ISSUE TITLE
Keeps logging me out

### ISSUE THREAD

**Issue by lukebrooker** [wshearn/openshift-cartridge-osbs#11]
I can login in fine and get the initial graph. But as soon as I click anything in the navigation I am logged out and asked to log in again.

**Comment by wshearn** [comment]
Sounds like passport is having issues with the session store. Is redis running on the osbs gear?

**Comment by lukebrooker** [comment]
For some reason I installed the redis cartridge with my app instead of osbs :/ whoops, sorry.

So that is fixed, but now I am getting "Add the client cartridge to some of your gears", which I am pretty sure I did correctly…

![screenshot 2015-01-05 09 53 12](https://cloud.githubusercontent.com/assets/1305265/5608063/bda6c37e-94c0-11e4-812a-e329ae451205.jpg)

If this is a problem I can open another issue?

**Comment by wshearn** [comment]
Try removing the client cartridge and readding it.

**Comment by lukebrooker** [comment]
Yeah I had tried that… and just tried again. Restarted both. Still doesn't work :/
